src/admin/resolvers/mutations/ApprovedItem.integration.ts

```diff
@@ -973,6 +973,10 @@ describe('mutations: ApprovedItem', () => {
       );
       expect(approvedItem.url).to.equal(input.url);
       expect(approvedItem.imageUrl).to.match(urlPattern);
+      expect(approvedItem.externalId).is.not.undefined;
```

Comment:
this is the actual change, added assertions for fields we expect in the backfill code